src/xcode/ENA/ENA/Source/Services/OTP/__tests__/OTPServiceTests.swift

```diff
@@ -5,6 +5,7 @@
 import XCTest
 @testable import ENA
 
+// swiftlint:disable:next type_body_length
```

Comment:
Why is the linter touching this file? It should be ignored 🤨

Reply:
I don't really know but linter is warning otherwise. So I will let it as it is